src/app/Profile/dress/page.tsx

```diff
@@ -0,0 +1,299 @@
+
+    onMouseDown={(e) => {
+      const container = e.currentTarget;
+      let isDown = true;
+      let startX = e.pageX - container.offsetLeft;
+      let scrollLeft = container.scrollLeft;
+
+      container.style.cursor = "grabbing";
+
+      const handleMouseMove = (moveEvent: MouseEvent) => {
+        if (!isDown) return;
+        moveEvent.preventDefault();
+        const x = moveEvent.pageX - container.offsetLeft;
+        const walk = x - startX; // Distance scrolled
+        container.scrollLeft = scrollLeft - walk;
+      };
+
+      const handleMouseUp = () => {
+        isDown = false;
+        container.style.cursor = "grab";
+        document.removeEventListener("mousemove", handleMouseMove);
+        document.removeEventListener("mouseup", handleMouseUp);
+      };
+
+      document.addEventListener("mousemove", handleMouseMove);
+      document.addEventListener("mouseup", handleMouseUp);
+    }}
+  >
+      {tabs.map((tab) => (
+        <button
+          key={tab.id}
+          onClick={() => handleChangeTab(tab.id)}
+          className={`w-20 h-7 flex flex-shrink-0 items-center justify-center rounded-lg font-medium
+            ${activeTab === tab.id
+              ? "bg-gradient-to-b from-[#ECEFF6] to-[#3B83A9] text-[#262D37]" // Style for active tab
+              : "bg-gradient-to-b from-[#092B44] via-[#625B87] to-[#D2CAFF]  text-white " // Style for inactive tab
+            }`}
+        >
+          {tab.label}
+        </button>
+      ))}
+      
+    </div>
+  );
+};
+
+type element = {
+  id: string;
+  imageSrc: string;
+  isLocked: boolean;
+  colors: number;
+}
+
+type item = element[]
+
+
+function ItemsGrid({ items, activeTab, activeItemId, setActiveItemId, handlePartSelection }: { items: Record<string, item>, activeTab: string,
+  activeItemId: string|null, setActiveItemId: (tab:string|null) => void, handlePartSelection: (part:string, id:string) => void }) {
+
+  const displayedItems = (() => {
+    if (!activeItemId) {
+      return items[activeTab];
+    }
+
+    const activeItem = items[activeTab]?.find((item) => item.id === activeItemId);
+
+    if (activeItem && activeItem.colors > 1) {
+      return Array.from({ length: activeItem.colors }, (_, index) => ({
+        ...activeItem,
+        id: `${activeItem.id}-${index + 1}`,
+        imageSrc: `/model/${activeTab}/${activeItem.id}/${activeItem.id}-${index + 1}.webp`,
+      }));
+    }
+
+    return items[activeTab];
+  })();
+
+  const handleClickItem = (id: string, imgSrc: string) => {
+    setActiveItemId(id);
+    handlePartSelection(activeTab, imgSrc);
+  }
+
+  return (
+    <div className="grid grid-cols-3 w-full justify-center justify-items-center bg-[#ECF0F6]/80 h-60 rounded-b-lg"
+        style={{
+        overflow: "auto", 
+        scrollbarWidth: "none", 
+        msOverflowStyle: "none", 
+        WebkitOverflowScrolling: "touch", 
+        overflowY: "scroll"
+      }}>
+      {displayedItems && displayedItems.map((item: element) => ( 
```

Comment:
error: Prefer using an optional chain expression instead, as it's more concise and easier to read.  @typescript-eslint/prefer-optional-chain

solve: change `{displayedItems && displayedItems.map((item: element) => ( ` to  ` {displayedItems?.map((item: element) => ( ` using `?` to check component is valid

more information: https://typescript-eslint.io/rules/prefer-optional-chain/